tests/workload/pag.robot

```diff
@@ -8,7 +8,7 @@ Library           String
 Library           OpenAFSLibrary
 
 *** Variables ***
-${PRINT_GROUPS}    python -c 'import os; print os.getgroups()'
+${PRINT_GROUPS}    python -c 'import os; print(os.getgroups())'
```

Comment:
Good catch. Instead of print, could we just change this to use sys.stdout.write(), which is the same everywhere?

Reply:
I think the print function  is okay.  It works with python2.7 or python3 .. and I just checked on a centos 6 system, it appears to also work with python 2.6.

Reply:
Just poked through the doc.  The print function was added in 2.6.

Reply:
Yes, but the print function requires the "from __future__", like jsbillings pointed out, to be available on py 2.7.

I'll just change this to use sys.write(), thanks!

Reply:
Hi Cheyenne, can you try 3a8c9133193edef177970bf4a0041907cb727d0b ?